Unfettered Hai side-story: Hai Slaves

NomadicVolcano opened the pull request:
This is replaced with #4943 

~~Please vote on the comment below (that begins with, "I call for a vote") on whether I should close this pull request.~~ This offer is rescinded.

The limits of Hai pacifism are tested after 900 Hai are captured by Human slavers. Captain `<last>` lacks a ground combat force, and must seek help from a government. Political reality forces the Republic, Syndicate, and Hai Elders to delay their help so long that the slaves may be sold first. Only one band of unlikely heroes remains: the Unfettered Hai.

They were the protectors of the Hai once before. Can they be so again? Is the divide among the Hai too great to cross? Or is there some hope of peaceful reconciliation?

Use this save file to test. Visit the spaceport; I've edited the save so the event has a 100% chance of happening.

[Omega Turtle~visit spaceport.txt](https://github.com/endless-sky/endless-sky/files/4309873/Omega.Turtle.visit.spaceport.txt)

Your flagship is a cloaking Bactrian. There are many options for combat ships, but most are capital ships. There is combat late in this story, so you may want to switch flagships when you are asked to escort Centipedes.

This story was inspired by an unpublished story by @Arachi-Lover he showed me as a story idea for my quest to improve the storytelling quality of the Hai. The new story bears little resemblance to the original, and apparently horrifies him.

Notes:

1. The three Shield Beetles sent by the Unfettered Hai are equipped extremely well, but they can only defeat the Korath and Pirates without help about a third of the time. If you don't feel like testing the battle, bring your seven KIV 349s and win in a few seconds.

2. Notice that you are on good terms with the Unfettered Hai after this story. That means, if you have other missions that require fighting them, you have to be the dishonorable one. They won't attack an ally; they have a warrior's code. You have an agenda.

3. If you land at Darkcloak without 23 cargo space free, and you need to leave and come back, you should be able to. The authorities will scold you for taking too long, but they'll let you back in.

4. The Centipedes you escort are upgraded. I don't know if I made them too strong.

## PR Checklist
 - [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [X] I uploaded the necessary image, blend, and texture assets here: N/A
 - [X] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: N/A

Anarchist2 commented on data/governments.txt:
```diff
@@ -47,6 +47,21 @@ government "Bounty Hunter"
 	"hostile hail" "hostile bounty hunter"
 
+# Bounty hunters that the player's escorts will attack and vice-versa
+government " Bounty Hunter "
```
Not using `"display name"`?

NomadicVolcano replied:
They need specific attitudes for the missions to work, and no government presently has that attitude. If the government name, `" Bounty Hunter "` may cause problems, I can change it to something else and use `display name`.

NomadicVolcano replied:
I'll test that later today.

beccabunny replied:
What I think Anarchist meant is, no matter the name you give it, you should always give it the intended display name in order to display normally in game. See: https://github.com/endless-sky/endless-sky/issues/4767

NomadicVolcano replied:
The string, `" Bounty Hunter "` is too generic for the purpose the government serves. It is specifically there to hunt the player and the Hai, but nobody else. I'm going to give them a longer name like `Bounty Hunters that Fight Hai` and then use `display name` to make it show up as `Bounty Hunters`.

Arachi-Lover commented:
>the original story's spirit remains strong and will never be forgotten so long as this one holds its memory
:thonk:

>To benefit the greater good, the Hai Elders, Republic Parliament, and Syndicate refuse to intervene.
that's unnecessarily contrived and hard to believe.

it's one thing if Scar's Legion steals some technology, and keep their already unlikely-to-be-found hidden, but now 900 Hai are kidnapped and enslaved and ONLY the Unfettered will do anything? Wtf? Did I forget to play the game where the Republic Navy is incompetent and lazy, Syndicate-bribing-parliament means literally anything involving the Syndicate is outside the Republic's range, where Navy Intelligence doesn't exist, where the Hai simply flat out allow their people to be enslaved because of "trolley problem but in space" ?

NomadicVolcano commented:
> it's one thing if Scar's Legion steals some technology, and keep their already unlikely-to-be-found base hidden, but now 900 Hai are kidnapped and enslaved and ONLY the Unfettered will do anything? Wtf? Did I forget to play the game where the Republic Navy is incompetent and lazy, Syndicate-bribing-parliament means literally anything involving the Syndicate is outside the Republic's range, where Navy Intelligence doesn't exist, where the Hai simply flat out allow their people to be enslaved because of "trolley problem but in space" ?

Read the story before jumping to conclusions.

Also think back to what happens in the game up to this point.

The Republic Navy doesn't care about slaves, nor are they a morally-sound organization. When they scan slaves on your ship, they blow up the ship. Pirate vessels are crewed by slaves, and the Navy makes no attempt to free them. They blow up the ship. Pirate planets are covered with slaves, and the Navy makes no attempt to save them. This is the same Navy whose stance towards the Alphas is, "kill every last Alpha man, woman, and child, and mandate abortions of Alpha fetuses." The final solution is the only solution. When they bomb Haven, likely killing millions of people (including many slaves), they're more concerned by the political repercussions. Of course, they didn't have the courage to do it themselves; they trick you into doing mass murder for them. When the player calls them on their immoral behavior during the Free Worlds campaign, their excuse is, "we're just following Parliament's orders."

The Syndicate won't intervene because they will lose thousands of security personnel, making hiring too expensive. Plus, the slavers have connections to Syndicate executives, and are likely pulling strings behind the scenes. Adding a few hundred slaves to millions in the galaxy is a tiny cost compared to the loss of thousands of workers.

Parliament does not want to risk another bloody war after the last one ended, over the fate of a few slaves. The political situation is precarious, and society is already scarred.

The Hai are afraid of sparking a war against the Humans when they are barely able to hold back the Unfettered. They'll almost certainly lose, and end up as part of the Unfettered's empire. Millions of people will die for the sake of 900 slaves.

The Unfettered do not think about long-term consequences. They see a good fight that will make them heroes, and they take it.

NomadicVolcano commented:
To clarify matters, slaves are not people in Endless Sky. Nobody cares about them. You don't see people on non-Pirate planets complaining about millions of slaves and how they should be freed. Merchant's don't say, "It's terrible that the pirates are forcing slaves to fight me." They say, "When pirates attack, leave nothing to bury." When the Navy sees a disabled pirate ship, likely filled with slaves, they don't board and free the slaves. Then shoot the ship.

Seriously, expecting the Republic, Parliament, Navy, or Syndicate to sacrifice thousands of troops to free 900 slaves out of millions in the galaxy is extremely contrived. If you think they would do it, then you haven't been reading the text in the game. The reconciliation Free Worlds has sturdy morals and believes in self-sacrifice. They might do it, but not if they have to send warships across Republic and Syndicate space, attacking an operation that is run by Syndicate executives.

Also, consider what would happen if the Hai did send thousands of troops and dozens of ships into Human space. It would be nearly impossible to hide the Hai from the Human populace. They would have to go public. There would be a massive influx of refugees into Hai space. They would either have to close their borders or overwhelm their social welfare system. 

After the Pug attack, anti-alien hysteria will be at an all-time high. If you introduce another alien fleet invading Human space, everyone will unite again very quickly. Parliament will have to intervene. The military response will be swift and decisive, and inevitably destructive.

Any government involvement would be self-destructive.

I don't write stories without first carefully thinking about every aspect of how it interacts with the rest of the game. I thoroughly check the spaceport messages, hails, wikis, side-stories, and main stories to make sure my story is consistent. I also checked with people who have future plans for the Hai to make sure it leads well into their stories.

NomadicVolcano commented:
As for the number 900, do you really think Captain `<last>` is the only one getting these, "smuggle a Hai into Human space" missions? You're one of thousands of merchants flying through Hai space. There are ships flying through the wormhole several times a minute, and there are probably Hai on many of them.

The story tells of a group connected to Syndicate executives that want to create a market for alien slaves. They'll have no trouble tracking down a few hundred Hai over the course of several months. If they're careful about how the Hai vanish, then it will take several weeks at least for people to figure out what happened. Now that they have enough slaves for breeding, they don't need to take any more. They just need a durable enough ground facility to make it politically impossible for governments to intervene; and customers wise enough to keep their exotic slaves secret.

Hai crave excitement and new situations; the threat of pirates won't deter them. It will only encourage them. You can see this on Snowfeather, "it is a popular vacation spot for the Hai, who find the unpredictability and occasional violence of the weather here to be invigorating." The game often refers to their excitable nature and high level of energy. They like to try Human "prison cruises" as a new, but scary, experience. The Hai you bring to Human space realize the risk but are excited to try anyway. Nanachi is willing to fly her Heavy Shuttle past pirates that took down a Leviathan, even though she knows full well that she may be killed in the attempt.

It is more than plausible that 900 Hai would be captured; it is a miracle that it didn't happen sooner.

downRightCoder256 commented:
> slaves are not people in Endless Sky

just like in any period in history where mass slavery was present, wouldnt there be some sort of activist group opposing such idea? maybe get them involved

NomadicVolcano commented:
> > slaves are not people in Endless Sky
> 
> just like in any period in history where mass slavery was present, wouldnt there be some sort of activist group opposing such idea? maybe get them involved

Such a group is unlikely to have a large ground combat force, nor contact with the Hai. The best they could do is lobby parliament. If that could end slavery, it already would have. An effort to end slavery in Human space would be a large campaign that involves taking down the Alphas, and that is out-of-bounds until the mythical MZ returns.

Also, if you read the story (instead of just commenting) you'll see that it is not Captain `<last>` who decides to go to the Unfettered; Teeneep informs you that she will go to them. They're the protectors of the Hai, and in her mind, they're the only ones who can or will help. She isn't going to say, "Lets wander around Human space for months looking for help while the slavers prepare to save my people." She needs a solution urgently, and the Unfettered are very good at urgency. Lobbyists aren't.

downRightCoder256 commented:
well, i didnt mean for you to have THEM rescue te hai slaves. you could get them involved, just to show that such exist in ES. you could add in a small dialogue where they too start mustering some forces using contacts they have, shortly after the player visits the navy, and end up organizing themself to attack about the same time as the unfettered, but for the actual force they would end up bringing, to have nothing more powerful than a couple of argosies. annoying at worst, satirical at best (at least in my opinion)

NomadicVolcano commented:
> well, i didnt mean for you to have THEM rescue te hai slaves. you could get them involved, just to show that such exist in ES. you could add in a small dialogue where they too start mustering some forces using contacts they have, shortly after the player visits the navy, and end up organizing themself to attack about the same time as the unfettered, but for the actual force they would end up bringing, to have nothing more powerful than a couple of argosies. annoying at worst, satirical at best (at least in my opinion)

I intend for this to be a short side-story, so I don't want to add any more characters than are needed. The main point is to expand the story of the Hai, not the Human slave situation. 

An anti-slavery group would be a fascinating follow-on story though, especially if it was secretly funded by the Hai. (Or the Unfettered Hai.) I don't plan on writing such a thing, but it does sound interesting.

Arachi-Lover commented:
I did read the story. My point stands. The Navy is always shown as honorable in story missions, the ones in Farpoint especially, remember that they dislocated large portions of the paradise worlds fleet to evacuate Hope when the planet was having the collapse. Parliament might not care about regular slaves, but these are Hai, and would them and the Syndicate want to soil relationships and possible business by just shrugging off some human lawbreakers enslaving Hai people? Parliament doesn't want "another bloody war", so their plan of action is... pissing off the Hai by not stopping human pirates from enslaving them? Being corrupt doesn't mean being a complete imbecile.
 Having Hai *ships* so much so as be there for a rescue operation would have everyone immediately know of them by consequence? Truly? Then why is it that we didn't such a thing mentioned with the Osprey bounty with Hai outfits in the Free Worlds bounty hunting jobs? I imagine the god damn Hai would be way better at keeping themselves secret than some pirate who stole their outfits and is terrorizing the south, especially if the Navy helped them.
 "there are probably Hai in many of them" isn't a very strong argument. I can say that there are probably many human pilots that manage to steal Jump Drives from Kor Raiders, the Syndicate themselves did it, but apparently the player is the only one that has one, went to the coalition, and told them of the Hai. I mean surely, of the many humans with JDs, probably some of them would go to the Hai and later to the Coalition, no?

>After the Pug attack, anti-alien hysteria will be at an all-time high. If you introduce another alien fleet invading Human space, everyone will unite again very quickly. Parliament will have to intervene. The military response will be swift and decisive, and inevitably destructive.

I too remember of the time everyone united again very quickly against the Sestor fleet and how the military response was swift and decisive. Oh wait no it was just <first> <last> beating the fleet/bombing zenith.

>Hai crave excitement and new situations; the threat of pirates won't deter them. It will only encourage them. You can see this on Snowfeather, "it is a popular vacation spot for the Hai, who find the unpredictability and occasional violence of the weather here to be invigorating.
 
yeah lemme just flip that logic them. in the Hai's words, humans like the opposite of that, calm places, where they can relax. so why didn't everyone just chill and submit to the Pug instead of the chaotic fighting that ensued?

>Nanachi is willing to fly her Heavy Shuttle past pirates that took down a Leviathan, even though she knows full well that she may be killed in the attempt.

yes please do educate me on the character I wrote.

 I have indeed been reading the text in the game, for the past ~4 years in fact.

NomadicVolcano commented:
The Navy admirals that ordered the evacuation of Hope were replaced. "Honorable" is a word that can be misused. The Navy keeps their promises, enforces the law, and follows orders. It is clear that the Navy still doesn't patrol Syndicate space after the war. Either laws or orders prevent them. Apart from the incident at Hope, the Navy has always shown they will follow immoral orders, over their own conscience. Their conscience leaves much to be desired, as shown by a self-proclaimed Hindu pacifist (and possible Beta) who decides the only solution to Alpha fetuses is to force abortions. I'm sure a few million Hindu pacifists would take issue with that.

Parliament could probably be convinced to care after a few weeks of negotiation, but large parliamentary democracies are not know for their speed. The Hai don't have that long, and neither does Teeneep's patience.

I suppose I could incorporate that idea into the story if a more realistic Parliamentary response is justified.

Furthermore, the only evidence anyone has of the fate of the Hai is the claim of one Hai woman and a pirate.

NomadicVolcano commented:
I had a long discussion on Discord about changes to this story, primarily with Arachi, and this is my attempt at summarizing the discussion:

1. Navy goes to parliament who requests an investigation.
2. Teedeep doesn't want to wait, so she's sent to the Syndicate
3. The Hai elders, both times they're encountered, say they're still debating.
4. Teedeep doesn't want to wait for the Hai elders to debate. That's why she goes to the Unfettered.
5. Hai military folk show up during the raid at Stormhold. Perhaps a significant fraction of the ground combat force. They defected out of disgust for the Hai Elders' delay.

The missions themselves will not change as a result of this, so you can still play-test the battles. It only changes the text of the conversations.

downRightCoder256 commented:
maybe add somewhere between 4 and 5 a small dialogue telling something like “surveillance drones have gained intel confirming large numbers of hai kept in captivity on Stronghold”, because come on, there’s no way regular hai would mobilize anything over claims of a single person (or hai, that is)

NomadicVolcano commented:
> maybe add somewhere between 4 and 5 a small dialogue telling something like “surveillance drones have gained intel confirming large numbers of hai kept in captivity on Stronghold”, because come on, there’s no way regular hai would mobilize anything over claims of a single person (or hai, that is)

Actually, I wasn't going to do it for that exact reason you state. The Navy was planning on doing such a survey, and possibly send in intelligence agents too, but Teedeep wouldn't wait.

NomadicVolcano commented:
I have updated the story so it focuses more on the nature of slavery in Human space, and the questionable ways authorities deal with it. There are no plot holes, the characters are more realistic, and the Navy's response matches current lore, dialog, and game mechanics. Comments in `unfettered missions.txt` back up that claim. Republic Intelligence has its say, and claims to represent the interests of Parliament. 

The main character, Teeneep, is now a prominent civil rights activist among the Hai, has an even temper, and is a skilled politician. This explains why Hai authorities implicitly trust her word. She is unknown in Human space, so her word is questioned there. This sets her up as a pro-Unfettered, pro-change, and pro-ethics demagogue among the Hai, giving their society another moral dilemma to deal with in a follow-on story.

The Syndicate executive is far more understanding since his father and brother were kidnapped into slavery on pirate ships. Their fate at Navy hands was not kind.

Hai military officers do *not* turn traitor and join the Unfettered at Stormhold. They die by the thousands daily against Unfettered at Wah Ki. They would never turn traitor. I added a mission where they explain, in no uncertain terms, that they will not disobey the Hai elders. They have the foresight to know that path can lead to a military dictatorship. The Hai military is the bastion of stability, and the servant of the state, not its ethics instructor.

I have put everything in more of a moral gray area, giving strong justifications for even the most seemingly inhumane decisions.

NomadicVolcano commented:
I call for a vote. Thumbs down and I will close this pull request. Thumbs up and I'll keep it. Arachi's vote counts double. Put your thumbs on this comment. Once one thumb direction passes the other by seven (after doubling Arach's vote), I'll consider the vote complete. Otherwise, it is up to the code managers to decide.

NomadicVolcano commented:
> Otherwise, it is up to the code managers to decide.

By this I mean I'll go through the review process as normal if I don't close the PR.

NomadicVolcano commented:
I suspect some people don't like this story because they're not reading between the lines to see the politics that are forcing decisions. I've added annotations between the lines explaining what is happening.

Amazinite commented on data/hai/unfettered missions.txt:
```diff
@@ -0,0 +1,1183 @@
+			`	It is difficult to make out Hathor's words through the thick door, but you can gather the gist of his conversations. He talks to a few slavers throughout Human space and finds a group near Syndicate space that have hundreds of Hai. They agree to meet you, and Hathor asks for 100,000 credits payment if the two of you are taken as slaves.`
+			choice
+				`	(Continue.)`
+			`	Hathor emerges from his office. "I've scoured the galaxy, and only one slaver has Hai, but they have plenty. I'll want 50,000 credits before I give you details and a clearance code."`
```
Didn't I just hear where these slavers were by spying on Hathor? We at least know that they're near Syndicate space. That narrows down our search to 4 planets. We could decline to pay and search on our own.

ravenshining commented on data/hai/unfettered missions.txt:
```diff
@@ -0,0 +1,1183 @@
+		has "Hai Slave Rescue [14]: done"
+	on offer
+		conversation
+			`The Hai prepare their nine centipedes for the trip to <planet>. Hopefully there won't be a pirate ambush waiting; you have no Unfettered helping you this time.`
```
Why on earth would the unfettered NOT want to make sure the transports are safe?
```suggestion
			`The Hai prepare their nine centipedes for the trip to <planet>. Hopefully the pirates won't mount a counterattack.`
```

ravenshining commented on data/hai/unfettered missions.txt:
```diff
@@ -0,0 +1,1183 @@
+		personality disables plunders opportunistic harvests staying target
+		fleet "Large Core Pirates" 2
+		fleet "Small Core Pirates" 5
+
```
The Geocoris are busy, and the other two shield beetles may nor may not still survive, but at least this one should still be around.
```suggestion
	npc staying save
		government "Heroic Unfettered"
		system Alcyone
		personality heroic escort disables plunders
		ship "Unfettered Flagship" "Admiral Choot'k"

```

NomadicVolcano replied:
The admiral is busy ransacking the planet. 

Also, he doesn't expect you to have trouble. You escorted the centipedes all the way from Hai space to Canyon. Now you're just escorting them one system away. As long as your flagship isn't a Bulk Freighter, you should be able to land before the ships are damaged. These aren't normal Centipedes; they have Hai Diamond Regenerators and extra ammo. They're designed for combat.

NomadicVolcano commented on data/hai/unfettered missions.txt:
```diff
@@ -0,0 +1,1183 @@
+			choice
+				`	(Continue.)`
+			`	Agent Kopkow says, "That does concern me. Aliens with advanced technology in the hands of Syndicate extremists?"`
+			`	Admiral Cain nods in agreement, "I'll ask Navy Intelligence to confirm your story. We'll send agents and reconnaissance drones. If they prove your words true, I'm sure I can convince my superiors to take the base intact. We'll have to interrogate the surviving slaves, and press charges against any collaborators. I'm sure the vast majority will be allowed to return home."`
```
Done. I'm getting errors though, but I'll fix those when I finish merging changes.

NomadicVolcano commented on data/hai/unfettered missions.txt:
```diff
@@ -0,0 +1,1183 @@
+		system "Ultima Thule"
+		fleet "Large Northern Pirates" 2
+		fleet "Small Northern Pirates" 4
+
```
Actually, they only use 28 ramscoops: 25 Geocorises and 3 Shield Beetles. I reduced the requirement to 28.

NomadicVolcano commented on data/hai/unfettered missions.txt:
```diff
@@ -0,0 +1,1183 @@
+			`	"Surely the fearless hero <first> <last> will help?"`
+			choice
+				`	"Of course!"`
+				`	"No, I'm late for a pedicure."`
```
I'm replacing this mission, so the change is no longer relevant.

NomadicVolcano commented:
My updates to this branch will be a near-total rewrite, so I'll probably close this PR and open a new one in a few days. I'm going to leave this PR open so people can comment on the story as it is. Many of the characters will be re-used.

There is a more complex approach to the Hai Slaves story that I could have taken, but didn't because, frankly, I don't find the idea of a Hai Slave rescue that exciting. I think Arachi's story did a better job. I spent a lot more time on my other stories. Now that there is some interest in the story, and some intense loathing of it, I'll finish it properly.

The better approach involves adding disconnected missions hinting about the story before it happens; adding some new systems, having the Navy and Syndicate raid two different facilities, and forcing the use of the Unfettered because they're the only ones who have Quantum Keystones, many Jump Drives, and a strong ground combat force. This also involves resurrecting Arachi's approach to Hai slave captures, plus a few more.

So yes, a lot more work, but I'll do it this time.

NomadicVolcano commented:
This is replaced with a new PR: https://github.com/endless-sky/endless-sky/pull/4943
